examples/ConsoleApp/ConsoleApp-Charles.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What does this .csproj do? Was it included by accident?

Reply:
Oh, I mis-uploaded my test application. I will remove it.